data/games/under_night_in_birth_exe_late_st/images/hyde/index.js

```diff
@@ -1,6 +1,134 @@
 export default [
-  // {
-  //   "imageId": "5A",
-  //   "image": require('./5A.png')
-  // }
+  {
+    "imageId": "5A",
+    "image": require('./5A.png')
+  },
+  {
+    "imageId": "2A",
+    "image": require('./2A.png')
+  },
+  {
+    "imageId": "5B",
+    "image": require('./5B.png')
+  },
+  {
+    "imageId": "2B",
+    "image": require('./2B.png')
+  },
+  {
+    "imageId": "5C",
+    "image": require('./5C.png')
+  },
+  {
+    "imageId": "5C_charge",
+    "image": require('./5C_charge.png')
+  },
+  {
+    "imageId": "2C",
+    "image": require('./2C.png')
+  },
+  {
+    "imageId": "jA",
+    "image": require('./jA.png')
+  },
+  {
+    "imageId": "jB",
+    "image": require('./jB.png')
+  },
+  {
+    "imageId": "jC",
+    "image": require('./jC.png')
+  },
+  {
+    "imageId": "jC_charge",
+    "image": require('./jC_charge.png')
```

Comment:
It looks like `jC.png` and `jC_charge.png` are the exact same image. We could save a bit of space by just referencing the same image. 

```
$ diff -s jC.png jC_charge.png
Files jC.png and jC_charge.png are identical
```

Reply:
This thought had entered my mind. For some reason, I thought that having a separate, identical image would make it easier to replace the image later if we wanted to.

That doesn't make any sense, though, so we should definitely reference the same image.